releasenotes/releasenotes_v5.rst

```diff
@@ -3,6 +3,10 @@
 Current Firely Server release notes (v5.x)
 ==========================================
 
+For information on how to upgrade, please have a look at our documentation on :ref:`upgrade`. You can download the binaries of the latest version from `this site <https://downloads.fire.ly/firely-server/versions/>`_, or pull the latest docker image::
```

Comment:
This should be a “note” for better visibility. Do you prefer to add the upgrade links here or should we just add the link to the other page?

Reply:
With other page, you mean a link to the release notes from the upgrade page? Because that link already exists. I will make a note of this message :)